### PR TITLE
Add S-PWQFN-N16 with 1.7x1.7 EP

### PR DESCRIPTION
**Pull request by zonque**

### CODE REVIEW

**Comment by zonque** [comment]
Could I get a review on this one? I have a symbol pending that depends on this footprint.

**Comment by chschlue** [comment]
Of course! Sooner or later. Probably later.

There are 122 open PRs and yours is four (!) days old.
Also, you couldn't even be bothered to reference those pending PRs.

Hint: Not creating additional work for reviewers is among the best ways to get a review in time.

**Comment by zonque** [comment]
Oh, my apologies. I wasn't aware of the backlog, and I didn't mean to be pushy. The symbol is not pending as a PR yet. Will post as soon as the footprint is merged. Take your time, thanks for your work, and sorry again!

**Comment by chschlue** [comment]
That's not how it works.
Please open all related PRs cross-reference them.

File: scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn_texas.yaml [hunk 1]
```diff
@@ -159,6 +159,57 @@ Texas_S-PWQFN-N16_EP:
...
+  lead_len:
+    minimum: 0.3
+    maximum: 0.5
+
+  EP_size_x:
+    nominal: 1.7
+    tolerance: 0.1
+  EP_size_y:
+    nominal: 1.7
+    tolerance: 0.1
```
Comment (chschlue) [inline scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn_texas.yaml:190]: Please use the package pad dimensions of 1.66mm for `EP_size_[xy]` and add `EP_size_[xy]_overwrite: 1.7` to get the suggested EP land size.

File: scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn_texas.yaml [hunk 2]
```diff
@@ -159,6 +159,57 @@ Texas_S-PWQFN-N16_EP:
...
+  # EP_paste_coverage: 0.65
+  EP_num_paste_pads: [2, 2]
+
+  thermal_vias:
+    count: [2, 2]
+    drill: 0.3
+    # min_annular_ring: 0.15
+    paste_via_clearance: 0.1
+    EP_paste_coverage: 0.5
```
Comment (chschlue) [inline scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn_texas.yaml:199]: Please remove to use script defaults.

**Review by chschlue — CHANGES_REQUESTED** [review]
There is nothing special about this QFN footprint.
Please use its default name of `WQFN-16-1EP_3x3mm_EP1.7x1.7mm` and move the size definition to https://github.com/pointhi/kicad-footprint-generator/blob/master/scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/wqfn.yaml

**Comment by zonque** [comment]
Thanks for the feedback. While adding moving it to the other location, I realized there is a definition already that serves the need quite well. Don't know why I didn't see that before. Sorry for the noise, I'll just go for the already existing option then.

**Pull request closed without merging**